app/models/views/applikation/result.rb

```diff
@@ -0,0 +1,28 @@
+module Views
+  module Applikation
+    class Result < Views::ApplicationResult
```

Comment:
I'm wondering if it's not confusing which class has responsibility for which part, we have `Views::ApplicationResult` and then `Views::Application::Result`. I think I understand the difference, but it's not obvious from those names and I think it'll confuse us later. 

What about renaming the `Views::AppicationResult` to `Views::ApplicationDecision` in preparation to the decision setup we'll be adding later?